### ISSUE TITLE
Transfer stuck with "resume token not found on server"

### ISSUE THREAD

**Issue by soupdiver** [zrepl/zrepl#644]
I have a `push` job on one side and `sink` on the other.
My push job is stuck with the following error:
```
Replication:
zdata/pve/backups/vms PLANNING-ERROR (step 0/0, 0 B/0 B) resume token `toguid` = 12215164771134439639 not found on sender (`toname` = "zdata/pve/backups/vms@zrepl_20221005_082605_000")
```

The mentioned snapshot does not exist anoymore. It was removed external from zrepl
I'm not entirely sure what the resume token is but is there a way to just restart the transfer/job and not resume if that said token can not be found?

Happy to provide more details if needed.

```
zrepl version
client: zrepl version=v0.5.0 go=go1.17.6 GOOS=linux GOARCH=amd64 Compiler=gc
server: zrepl version=v0.5.0 go=go1.17.6 GOOS=linux GOARCH=amd64 Compiler=gc
```

**Comment by problame** [comment]
The resume token is a ZFS primitive that zrepl uses to guarantee that replication makes forward progress in the presence of network failures, system outage, etc. Learn more here: https://zrepl.github.io/configuration/overview.html#how-replication-works

With default settings, your problem cannot happen unless something deleted the snapshot `zdata/pve/backups/vms@zrepl_20221005_082605_000` on the sending side (`push`).
Did you do any such thing? Or is there a Proxmox service that does that (assuming that datasets beneath `zdata/pve` are managed by proxmox)_

Please provide the `push` side configuration, secrets stripped.

**Comment by soupdiver** [comment]
> With default settings, your problem cannot happen unless something deleted the snapshot

yup, I wrote `The mentioned snapshot does not exist anoymore. It was removed external from zrepl`
So yea, the snapshot was indeed deleted. But now it's stuck with said error.
How do I continue with the next available snapshot if that one has been removed?

**Comment by problame** [comment]
Clear the partial recv state on the receiver dataset using `zfs recv -A`

**Comment by soupdiver** [comment]
> Clear the partial recv state on the receiver dataset using `zfs recv -A`

nice! seems that did the trick
thanks :)
